### ISSUE TITLE
About the details of training log

### ISSUE THREAD

**Issue by wilxy** [LuizScarlet/StableCodec#9]
Hi @LuizScarlet . Thank you for open-sourcing the training log; they are very helpful for reproducing your work. I have a couple of questions:

1. In the training log for stage 1, is the evaluation settings used for Kodak for real-time testing consistent with your final test script? 
2. Regarding the perceptual loss, which library did you use?
3. What are the details of using gradient_accumulation? Such as batch per GPU is 1 with two GPUs, and the gradient_accumulation is 4.

**Comment by LuizScarlet** [comment]
Hi @wilxy . Thanks for your interest! For your questions:

1. Yes, we use Kodak for evaluation during training, and the evaluation settings are almost the same with the final test script.
2. The perceptual loss for training (I guess you mean LPIPS?) uses `from torchmetrics.image import LearnedPerceptualImagePatchSimilarity`. For evaluation, please refer to `src/evaluate.py`, which is extended from [MS-ILLM](https://github.com/facebookresearch/NeuralCompression/blob/main/projects/illm/eval_folder_example.py). I would suggest fixing your library for image quality assessment, as different libraries may produce inconsistent results even when using the same metric.
3. Yes, we use 2 RTX 3090 GPUs with batch per GPU = 1 and gradient accumulation steps = 4.
